### ISSUE TITLE
Flow Mode

### ISSUE THREAD

**Issue by kennie-29** [Zy0x/Chimera#1]
the only option that is working is **BASIC,** the rest **FILE NOT FOUND**

**Comment by Zy0x** [comment]
Thanks for using this module

Flow mode only for Balance (AI) mode, but it is under improvement for more practical use on termux

**Comment by kennie-29** [comment]
Your module is the **best** **one** I've tried so far among all the modules.

**Comment by Zy0x** [comment]
> Your module is the **best** **one** I've tried so far among all the modules.

That's great to hear, and now it's fixed and fully usable

**Comment by kennie-29** [comment]
I'm back again, and it took me too long. There seems to be a bug that prevents me from opening any apps, even Magisk. I don't remember what this feature is called, but it's the one that disables all unwanted apps when you enter a game, which is great because it stops background processes. The problem is that I've become stuck in this mode. Even after restarting my device, it remains in this mode. I can only open the listed game; everything else, including Magisk, is disabled. As a result, I had to format my device.

**Comment by Zy0x** [comment]
> I'm back again, and it took me too long. There seems to be a bug that prevents me from opening any apps, even Magisk. I don't remember what this feature is called, but it's the one that disables all unwanted apps when you enter a game, which is great because it stops background processes. The problem is that I've become stuck in this mode. Even after restarting my device, it remains in this mode. I can only open the listed game; everything else, including Magisk, is disabled. As a result, I had to format my device.

You are using Eterna feature, it seems that some roms are not compatible with this feature. Can you tell us what rom you are using and what version in detail? basically you don't need to format the data, because you can use termux and then disable the feature. But in the future I will fix it to make it easier to use. For the time being, do not use the Eterna feature to be safer, just use the Flow feature. Sorry and thank you

**Comment by kennie-29** [comment]
any update?

**Comment by Zy0x** [comment]
> any update?

Yes of course, but it will take some time for version compatibility. If you want to see some changes check out my project "Lynx" for beta updates, in general it will be like that on Mediatek devices.
